### DISCUSSION TITLE
How to make `order` optional with Moor file query?

### DISCUSSION THREAD

**Discussion by CDDelta** [simolus3/drift#993] [Q&A] [answered]
I have this query defined in a `.moor` file. How do I make the `$order` term optional for callers?

```sql
allDrives ($order):
    SELECT * FROM drives
    ORDER BY $order;
```

I've tried making `$order = NULL` but get the error `This placeholder has a default value, which is only supported for expressions.`

**Comment by simolus3** [comment, marked as answer]
This is not currently supported, you'd have to pass something like `OrderBy([OrderingTerm.asc(Constant(1))])` to ignore the order by. I can look into default values for `ORDER BY` too, feel free to open an issue for that.
